src/client/search/searchReducer.js

```diff
@@ -184,11 +193,40 @@ const entityReducer = (state, action) => {
       details: action.details,
     };
   case RECEIVE_ENTITY:
+    if (state.entityType === TEAM_ENTITY) {
```

Comment:
Any chance of a comment quickly saying what this does and is for?
I think it's for finding the closest match to today right?